README.md

```diff
@@ -228,5 +282,27 @@ If those commands work you should be able to navigate to `http://localhost:8000/
 
 ## Testing
 We're an amazing project, so obviously we have tests :sparkles:  
-Make sure you have the requirements installed in your virtual environment, `cd` into `backend/uclapi` and then run :  
+Make sure you have the requirements installed in your virtual environment (and you have activated it) , `cd` into `backend/uclapi` and then run :  
 `python manage.py test --testrunner 'uclapi.custom_test_runner.NoDbTestRunner' --settings=uclapi.settings_mocked`
+
+## Our Custom Django Management Commands
+Whats that?! You want even more info? This section details any custom management commands we have created for django. You can view the full list of commands including the standard ones by running the command ```python manage.py --help``` and get more information on specific commands by running ```python manage.py command --help```. The most useful commands for development are listed below in addition to this however.
+
+### Triggering Webhooks Manually
+To trigger webhooks and test them you can run the command ```python manage.py trigger_webhooks``` with the optional flag of ```--debug``` to print details of each response. **Note: this will require having data in the room_booking database. You will also want to have done this atleast twice or else you will send hundreds of thousands of bookings to the endpoint**
+
+### Testing the Timetable System
+To test the timetables system you can run ```python manage.py test_personal_timetable```. This will query you for a ucl userid and then print out their timetable data in json format. **This will require you having populated both caches**
```

Comment:
Suggest this is phrased as follows:
```
This will prompt you for a UCL userid and then print out the corresponding timetable data in JSON format. **You will need to have populated both caches**
```